ADJ-Internship/BusinessService/BusinessService.csproj

```diff
@@ -17,4 +17,10 @@
     <ProjectReference Include="..\Repository\Repository.csproj" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Mvc.ViewFeatures">
```

Comment:
Don't reference MVC into Business Layer, Business layer should focus on business only